gradle/verification-metadata.xml

```diff
@@ -1681,6 +1681,14 @@
             <sha256 value="634075f04ebe719a681fc7a653e43900b0ff20104d6705d5852c1a6ec8b2cc83" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="io.netty" name="netty-all" version="4.2.4.Final">
+         <artifact name="netty-all-4.2.4.Final.jar">
+            <sha256 value="9ee9ba2f48b6becbecfb280187f58d18fdbb19fbeea286fc89dfcbdfeaec5d68" origin="Generated by Gradle"/>
+         </artifact>
+         <artifact name="netty-all-4.2.4.Final.pom">
+            <sha256 value="9dd848900f5cb2a23aafd3321f04fc757c49305c20cd41d17f559950cb77bdbf" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="io.netty" name="netty-bom" version="4.1.107.Final">
```

Comment:
Can the old netty versions be removed from the verification metadata?

Reply:
done in #9114